changelog/5173.breaking.rst

```diff
@@ -0,0 +1 @@
+Changed URL for the ` sunpy.net.dataretriever.sources.SRSClient` to a supported data archive.
```

Comment:
There seems to be more going on in this PR than just this change.

Reply:
This is all the PR does. 

Outside of adding support for any SRS files that are corrected.

Reply:
I am not sure this is a breaking change? The files are still largely the same right? Also it might be nice to just note here that it no longer downloads tarballs, just for completeness.

Reply:
I added it to be breaking since we can't guarantee the files are the same and I would rather label as an "extreme"

Reply:
I think in that case the entry needs to be a lot more detailed, saying why we moved, what we moved to and that we expect the files will be the same but they might not be. Breaking changes should give people all the info they need to see they (hopefully) aren't too bad or else it looks crappy on our changelogs, like we break everything all the time.

Reply:
K

Reply:
Have a look now.